src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudV2Spec.groovy

```diff
@@ -1191,8 +1191,10 @@ class FlowCrudV2Spec extends HealthCheckSpecification {
         withPool {
             involvedSwitchIds.eachParallel { SwitchId swId ->
                 with(northbound.validateSwitch(swId)) { validation ->
-                    validation.verifyRuleSectionsAreEmpty(["missing", "excess", "misconfigured"])
-                    validation.verifyMeterSectionsAreEmpty(["missing", "excess", "misconfigured"])
+                    Wrappers.wait(RULES_INSTALLATION_TIME) {
+                        validation.verifyRuleSectionsAreEmpty(["missing", "excess", "misconfigured"])
+                        validation.verifyMeterSectionsAreEmpty(["missing", "excess", "misconfigured"])
+                    }
```

Comment:
why? I don't really expect the flow to change its status(L1177) before all rule discrepancies are sorted out